style.css

```diff
@@ -570,6 +694,38 @@ html {
     z-index: 4;
   }
 
+  .modal {
+    width: 63%;
+    padding: 10px 20px;
+    border-radius: 16px;
+  }
```

Comment:
- [ ] You've done an excellent job styling your modal :+1:, but to align it even more closely with the Figma template, it would be beneficial to add additional bottom padding. This adjustment will create more space between the text and the edges of your popup, enhancing the overall visual balance and ensuring a more visually appealing presentation. Please consider implementing this change to improve the modal's alignment with the desired design aesthetics.

**Your design**

![Screenshot from 2023-05-16 12-25-27](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/105117832/7728a3e0-76c7-4dbb-b110-e27cfcb4dd3b)

**Figma**

![Screenshot from 2023-05-16 12-20-23](https://github.com/HtetWaiYan7191/HtetWaiYan7191.github.io/assets/105117832/230dfa26-e230-4d8d-b672-95c8f8ce5859)